js/viz/docs/docchartseries.js

```diff
@@ -1085,8 +1085,7 @@ var commonSeries = {
         /**
         * @name dxChartSeriesTypes_CommonSeries_aggregation_method
         * @publicName method
-        * @type string
-        * @acceptValues 'avg'|'count'|'max'|'min'|'ohlc'|'range'|'sum'|'custom'
+        * @type ChartCommonSeriesAggregationMethod
```

Comment:
Use the "Enums." prefix.
See also @type Enums.DashStyle

Reply:
fixed